oga/frontend/src/App.js

```diff
@@ -33,55 +36,58 @@ if ("serviceWorker" in navigator && "PushManager" in window) {
 }
 
 function App(props) {
-    let session = true;
-    props.isLoggedIn(); //sets state's authenticate
-    if (props.auth !== null)
-        return (
-            <ConnectedRouter history={props.history}>
-                <div className="App">
-                    <Switch>
-                        <Route path="/signup" exact component={Signup} />
-                        <Route path="/login" exact component={Login} />
-                        <PrivateRoute
-                            auth={props.auth}
-                            path="/main"
-                            exact
-                            component={Main}
-                        />
-                        <PrivateRoute
-                            auth={props.auth}
-                            path="/settings"
-                            exact
-                            component={Settings}
-                        />
-                        <PrivateRoute
-                            auth={props.auth}
-                            path="/ask"
-                            exact
-                            component={NewQuestion}
-                        />
-                        <PrivateRoute
-                            auth={props.auth}
-                            path="/map"
-                            exact
-                            component={Map}
-                        />
-                        <PrivateRoute
-                            auth={props.auth}
-                            path="/reply/:id"
-                            exact
-                            component={NewAnswer}
-                        />
-                        <Redirect exact from="/" to="/main" />
-                        <Route render={() => <h1>Not Found</h1>} />
-                    </Switch>
-                </div>
-            </ConnectedRouter>
-        );
-    else {
-        console.log("HERE");
-        return null;
-    }
+  let session = true;
```

Comment:
since session variable doesn't seem to be used anywhere in the class, perhaps we can remove this line?

Reply:
@taehioum  will remove it. I told him.